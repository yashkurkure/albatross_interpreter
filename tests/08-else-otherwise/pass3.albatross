while (a <> 0) {
    if (a==1) {
      return 1;
    } else {
      return 0;
    }
} otherwise {
    if (a<>1) {
      return a;
    } else {
      return b;
    }
}
