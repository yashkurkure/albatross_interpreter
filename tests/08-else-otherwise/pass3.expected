((a)<>(0))
((a)==(1))
(1)
(0)
((a)<>(1))
(a)
(b)

